src/main/java/org/tikv/common/PDClient.java

```diff
@@ -353,18 +354,21 @@ public List<Pdpb.Region> scanRegions(
     // introduce a warm-up timeout for ScanRegions requests
     PDGrpc.PDBlockingStub stub =
         getBlockingStub().withDeadlineAfter(conf.getWarmUpTimeout(), TimeUnit.MILLISECONDS);
+    org.apache.commons.lang3.tuple.Pair<ByteString, ByteString> range =
```

Comment:
Is it possible to normalize the `Pair` class we use? I don't get the point to use `org.apache.commons.lang3.tuple.Pair` instead of `org.tikv.common.util.Pair` or vice versa.

Reply:
OK.